app/models/course/assessment.rb

```diff
@@ -97,10 +98,15 @@ def assign_folder_attributes
 
   def set_defaults
     self.draft ||= true
```

Comment:
@weiqingtoh Can help change this ? It will overwrite the value if draft is `false`.